### PR TITLE
Add documentation for SecurityInfo.secretKeyLength

### PR DESCRIPTION
**Pull request by gregorypappas**
### Description

Add documentation for the SecurityInfo.secretKeyLength property.

### Motivation

Needs to be documented.

### Additional details

https://bugzilla.mozilla.org/show_bug.cgi?id=1778473

### Related issues and pull requests

Relates to mdn/browser-compat-data#18371

### CODE REVIEW

**Comment by gregorypappas** [comment]
@rebloor Hi, would you be able to review this? Thanks!

**Comment by rebloor** [comment]
@gregorypappas this duplicates my PR for the same change ([#23423](https://bugzilla.mozilla.org/show_bug.cgi?id=1778473)). If you would like to add the release note to this one, it might be quicker than waiting for a review on mine.

File: files/en-us/mozilla/add-ons/webextensions/api/webrequest/securityinfo/index.md [hunk 1]
```diff
@@ -67,6 +67,9 @@ Values of this type are objects. They contain the following properties:
     - "TLSv1.3"
     - "unknown" (if the version is not valid)
 
+- `secretKeyLength` {{optional_inline}}
+  - : `String`. The length (in bits) of the TLS secret key.
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/webrequest/securityinfo/index.md:71]: Are you sure it's a string? I'm sure when I checked, it said number
```suggestion
  - : `String`. The length of the secret key in bits.
```

**Review by rebloor — CHANGES_REQUESTED** [review]

**Comment by gregorypappas** [comment]
@rebloor Done.

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks @gregorypappas

**Pull request merged**